src/Cache/BlockEsiCache.php

```diff
@@ -53,19 +56,31 @@ class BlockEsiCache extends VarnishCache
      */
     protected $recorder;
 
+    /**
+     * @var ControllerResolverInterface
+     */
+    protected $resolver;
```

Comment:
```suggestion
    private $resolver;
```

Reply:
Why is this protected?

Reply:
Because the parent is: https://github.com/sonata-project/SonataCacheBundle/blob/3.x/src/Adapter/VarnishCache.php#L62

Reply:
Why does it even need to be declared here?

Reply:
I removed declaration of `$resolver` and `$argumentResolver`